src/scss/vendors/_vendors.scss

```diff
@@ -89,34 +120,6 @@
 	}
 }
 
-// To adapt to UIO high contrast themes
-[class^="gpii-ext-theme"],
-[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) {
-	// WeCount logo
-	.logo circle,
-	.logo path {
-		fill: none;
-		stroke: var(--logo-stroke, currentColor);
-		stroke-linecap: round;
-		stroke-width: rem(3);
-	}
-
-	// Search icon; IDRC, OCADU and Canada logos
-	header .site-nav-wrapper .site-nav .search-container svg {
-		fill: currentColor;
-	}
-
-	// Canada logo
-	--theme-link-fill: currentColor;
-
-	// Placeholder images on the "views" page
-	svg.news-item-img use {
-		stroke: var(--placeholder-stroke, currentColor);
-		stroke-linecap: round;
-		stroke-width: rem(3);
-	}
-}
-
 // For styles that require foreground and background colors from each theme
 @each $theme, $fColor, $bColor in $UIOThemes {
```

Comment:
It might be better to move this section to before the line 94 since this style belongs to the generic css outside of any specific media query breakpoint. After the change, the structure of this file will be more structured with the media query breakpoints.